tasks/main.yml

```diff
@@ -13,11 +13,21 @@
   with_items:
     - "{{ neofs_ir__data_dir }}"
 
-- name: Ensure configuration directory exists
+- name: Create configuration directory parent
   ansible.builtin.file:
     state: directory
     path: "{{ item }}"
```

Comment:
No `realpath | dirname` filter here?

Reply:
My fault, fixed, thank you for your attentiveness (: